src/main/resources/psu.json

```diff
@@ -5,5 +5,13 @@
 		"formfactor": "atx",
 		"power": 850,
 		"price": 159.94
+	},
+
+	"Prince of Pereira": {
+		"brand": "Matthew",
+		"efficiency": "Gold",
+		"formfactor": "atx",
+		"power": 9000,
+		"price": 9000
```

Comment:
weird component, I think we should change this

Reply:
ok, prince of pereira of matthew of james